File: 151002/Haiduk/RV/RV/Lab3/Lab3.Discussion/Models/Note.cs

using Cassandra.Mapping.Attributes;

namespace Lab3.Discussion.Models
{
    [System.ComponentModel.DataAnnotations.Schema.Table("tbl_Note")]
    public class Note
    {
        [ClusteringKey]
        public long Id { get; set; }
        [ClusteringKey]
        public long NewsId { get; set; }
        public string Content { get; set; }
    }
}

File: 151002/Zavaliuk/Lab1/discussion/src/main/java/by/bsuir/discussion/model/entity/Note.java
package by.bsuir.discussion.model.entity;


import lombok.Getter;
import lombok.Setter;
import org.springframework.data.cassandra.core.mapping.Column;
import org.springframework.data.cassandra.core.mapping.PrimaryKey;
import org.springframework.data.cassandra.core.mapping.Table;

@Getter
@Setter
@Table("tbl_note")
public class Note {
    @PrimaryKey
    private NoteKey key;

    @Column("content")
    private String content;
}